markdown/en/api/ff-asn.api.md

```diff
@@ -17,12 +17,12 @@ ___
 | Name | Description |
 | ---- | ----------- |
 | **opened**&nbsp;(Boolean) | Determines if the filter group is expanded. If you want to have a group opened by default: `<ff-asn-group opened>` does the trick. |
-| **collapsible**&nbsp;(String) **Options**:&nbsp;true,&nbsp;false (default: true) | Determines, if the filter group is collapsible. |
-| **for-group**&nbsp;(String) | Determines, which filter group the template should be applied to. If all is selected, then the template is applied to all groups if no other template suits the requirements. |
+| **collapsible**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "true") | Determines, if the filter group is collapsible. |
+| **for-group**&nbsp;(String) | Determines, which filter group the template should be applied to. If `all` is selected, then the template is applied to all groups if no other template suits the requirements. |
```

Comment:
~~If `all` is selected~~ If its value is not set or is `"all"`

Reply:
[same as here](https://github.com/FACT-Finder-Web-components/documentation/pull/108#discussion_r225253191)